recipes/pyagg/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.0.1" %}
+{% set sha256 = "30a1c5519696ff868104cfe7d33c8ec52e8698a8be6e21a3b6eb2d0c168f28aa" %}
+
+package:
+  name: pyagg
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/jwiggins/pyagg/archive/{{ version }}.tar.gz
```

Comment:
Is it OK to start with the fork and change it later? There hasn't actually been an official release of this library yet. I'm setting up CI right now and thought this recipe might be useful.

If I'm abusing conda-forge, just let me know. This isn't urgent :wink: